### PR TITLE
HIP 129: Carrier Beta Default Opt-In

### PR DESCRIPTION
**Pull request by zer0tweets**
Submitting the HIP re: Carrier Beta Default Opt-in per discussion from last MWG

### CODE REVIEW

File: HIP XXX: Carrier Beta Default Opt-In [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Any carrier with 5 million or more active subscribers can join the beta carrier program without the need for a community vote to become a service provider.
+- Beta carriers can unilaterally choose where and how much they pay for Wi-Fi data offload, including the ability not to pay at all.
+- However, beta carriers only get access to a limited number of hotspots that are opted in to service beta carriers.
+- Upon approval of this HIP, all Wi-Fi hotspots will be opted in to service beta carriers. Any new Wi-Fi hotspots that come online will also be opted in to service beta carriers by default.
+- Hotspot operators can opt in or out of servicing beta carriers in the builder app and dashboard at any time. Hotspot operators will also see data on which carriers are connecting to them each day and how much data they are using in the app and dashboard, allowing them to decide whether to stay opted in.
+- The network planner will add a dashboard showing hotspot ranking based on unique connections served during the trailing seven days, plus POC and data earnings for said hotspots. Hotspots that earn a lot of POC but serve no beta carriers (and vice versa) will be publicly visible.
```
Comment (abhay) [inline HIP XXX: Carrier Beta Default Opt-In:35]:
```suggestion
- The Helium Mobile Network Planner will add a dashboard showing Hotspot ranking based on unique connections served during the trailing seven days, plus Proof-of-Coverage and data earnings for said Hotspots. Hotspots that earn a lot of POC rewards but serve no Beta Carriers (and vice versa) will be publicly visible.
```

File: HIP XXX: Carrier Beta Default Opt-In [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- This HIP proposes no changes to POC or data pricing for now, but the dashboard will provide public visibility into how the beta is progressing across carriers.
+
+##Drawbacks
+
+Some hotspot operators who are not actively following the network may not realize that their hotspot is opted in to service free data traffic by default and may be caught by surprise. To mitigate this, Wi-Fi hotspot manufacturers will provide ample warnings through various means (notifications on the dashboard, emails, etc.) to ensure that everyone who wants to opt out of the beta has the ability to do so.
```
Comment (abhay) [inline HIP XXX: Carrier Beta Default Opt-In:40]:
```suggestion
Some Hotspot operators who are not actively following the network may not realize that their Hotspot is opted in to service free data traffic by default and may be caught by surprise. To mitigate this, Wi-Fi Hotspot manufacturers will provide ample warnings through various means (notifications on the dashboard, emails, etc.) to ensure that everyone who wants to opt out of the beta has the ability to do so.
```

File: HIP XXX: Carrier Beta Default Opt-In [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Hotspots that serve a lot of beta carriers for free and those that opt out will earn the same POC and data rewards. To mitigate this, it is suggested to display a public dashboard that will praise beta participants and shame the “reward squatters,” laying the groundwork for an eventual POC change, if required.
```
Comment (abhay) [inline HIP XXX: Carrier Beta Default Opt-In:42]:
```suggestion
Hotspots that serve a lot of Beta Carriers for free and those that opt out will earn the same POC and data rewards. To mitigate this, it is suggested to display a public dashboard that will praise beta participants and shame the “reward squatters,” laying the groundwork for an eventual POC change, if required.
```

File: HIP XXX: Carrier Beta Default Opt-In [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+##Rationale and Alternatives
+
+One alternative is to attempt to proceed with HIP 126, as originally planned, and change the pricing scheme for the entire network, allowing hotspot operators to dynamically adjust pricing. This change, however, is harder to get community consensus behind and harder to implement.
```
Comment (abhay) [inline HIP XXX: Carrier Beta Default Opt-In:46]:
```suggestion
One alternative is to attempt to proceed with [HIP 126](0126-flexible-data-pricing-for-mobile-network.md), as originally planned, and change the pricing scheme for the entire network, allowing Hotspot operators to dynamically adjust pricing. This change, however, is harder to get community consensus behind and harder to implement.
```

File: HIP XXX: Carrier Beta Default Opt-In [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+##Unresolved Questions
+
+Some community members have voiced the opinion that limits to data access should be implemented along with this change, such as throttling the data to a certain level and/or setting upper limits on the amount of data that can be used by beta carriers. Implementing such throttling will take time. We, therefore, suggest that this be implemented at a later stage when and if it becomes clear that such controls are required.
```
Comment (abhay) [inline HIP XXX: Carrier Beta Default Opt-In:50]:
```suggestion
Some community members have voiced the opinion that limits to data access should be implemented along with this change, such as throttling the data to a certain level and/or setting upper limits on the amount of data that can be used by Beta Carriers. Implementing such throttling will take time. We, therefore, suggest that this be implemented at a later stage when and if it becomes clear that such controls are required.
```

**Review by abhay — COMMENTED** [review]
several, mostly minor, nits. Feel free to accept

**Comment by abhay** [comment]
> Foundation, please assign zer0tweets a HIP number and put this simple HIP up for vote asap. Ready to go based on working group call Thurs.

waiting on hearing from @zer0tweets on my feedback.

**Comment by zer0tweets** [comment]
Ready to go form my end. All commits approved. Thank you for fixing my typos =)

**Pull request merged**